source/_others/2015-09-11-gray-hack.md

```diff
@@ -3,16 +3,26 @@ categories:
     - hack
 tags:
     - gray
-sponsor: unknown
-reverse: unknown
+sponsor: Facebook
+reverse: Hack logo
 photos:
     -
         file: gray-hack-0-400x300.jpg
         link: https://www.flickr.com/photos/adamculp/24834461270/
         credit: Adam Culp
+    -
+        file: gray-hack-1-400x300.jpg
+        link: https://www.flickr.com/photos/atomictaco/45898655611/
+        credit: Tim Bond
 ---
 Produced by the HHVM/Hack team at Facebook. It has not been released
-publicly. It has an HHVM logo in place of the PHP logo.
+publicly. It has a HHVM logo in place of the PHP logo. Three prototypes have
+been produced followed by 500 which will not be available for sale.
 
 The pHackyderm has several features that distinguish it from an elephpant including
 an articulated trunk, robotic eye, and boots.
+
+The first prototype is named Steve, and the second Steven. All have the HHVM
```

Comment:
A *bit* late to this PR, but FYI the second prototype was named Steph.  Steven is the name of the third and final prototype.

Reply:
Corrections like this are never too late!